src/coreclr/jit/llvmlower.cpp

```diff
@@ -670,6 +670,12 @@ void Llvm::lowerLocal(GenTreeLclVarCommon* node)
     }
 }
 
+static bool isFullDefStore(GenTreeLclVarCommon* lclVar, LclVarDsc* varDsc)
```

Comment:
I don't think there is a need for the new helper: `IsPartialLclFld` will suffice.

Reply:
I have a problem with `IsPartialLclFld` when `TypeIs(TYP_STRUCT)` as the size comparison is different.  Can struct stores be `GTF_VAR_USEASG` I suppose they can?

Reply:
Yes, they can.

Did you also change the code in `ssabuilder.cpp` to use `IsPartialLclFld` (they have to stay in sync)?

Reply:
No I did not, thanks!  Lets see if it goes green.